### PR TITLE
fix(firstMile): show sample app for python only

### LINKED ISSUE #4251: firstMile: Wizard Step 8: Update Finish Elements for Python, Javascript, and Go (What are the users' next steps?)
## Figma

See new Figma from @AyeAndy: https://www.figma.com/file/JfGYCOb7ZyQ8JHSqeu4fma/Onboarding-Exploration?node-id=714%3A2554)

## ACs

- The finish page has a micro-survey (thumbs up / thumbs down question)
- The finish page has the following boxes as optional next steps for the user:
           - Sample App
           - Key Concepts
           - InfluxDB University
- Add eventing for the sample app box, the key concepts box, and the InfluxDB University box. Make sure this data is tracked in amplitude per user

### PR DESCRIPTION
**Pull request by ChitlangeSahas**
Closes https://github.com/influxdata/ui/issues/4473
Closes #4251

Screen Shot 2022-04-25 at 10 22 12 AM

Screen Shot 2022-04-25 at 10 22 22 AM

Screen Shot 2022-04-25 at 10 22 04 AM

### CODE REVIEW

File: src/homepageExperience/components/steps/Finish.tsx [hunk 1]
```diff
@@ -73,6 +73,7 @@ export const Finish = (props: OwnProps) => {
     }
   }, [])
 
+  const showSampleApp = props.wizardEventName === 'pythonWizard'
```
Comment (hoorayimhelping) [inline src/homepageExperience/components/steps/Finish.tsx:76]: nice :+1:

File: src/homepageExperience/components/steps/Finish.tsx [hunk 2]
```diff
@@ -82,15 +83,17 @@ export const Finish = (props: OwnProps) => {
...
-          <p>
-            Play around with our template code of sample app to streamline your
-            own data into InfluxData.
-          </p>
-        </ResourceCard>
+        {showSampleApp && (
+          <ResourceCard className="homepage-wizard-next-steps">
+            <SafeBlankLink href="https://github.com/InfluxCommunity/sample-flask/blob/main/app.py">
+              <h4>{CodeTerminalIcon}Sample App</h4>
+            </SafeBlankLink>
```
Comment (hoorayimhelping) [inline src/homepageExperience/components/steps/Finish.tsx:90]: while you're in here, how about adding events for the links being clicked and then connecting this PR with https://github.com/influxdata/ui/issues/4251 by doing a `closes #4251`

Comment (ChitlangeSahas) [reply src/homepageExperience/components/steps/Finish.tsx:90]: Totally!

File: src/homepageExperience/components/steps/Finish.tsx [hunk 3]
```diff
@@ -82,22 +87,33 @@ export const Finish = (props: OwnProps) => {
...
+            <p>
+              Play around with our template code of sample app to streamline
+              your own data into InfluxData.
+            </p>
+          </ResourceCard>
+        )}
+        <ResourceCard
+          className="homepage-wizard-next-steps"
+          onClick={() => handleNextStepEvent('keyConcepts')}
```
Comment (ChitlangeSahas) [inline src/homepageExperience/components/steps/Finish.tsx:106]: this anon function seems to be the only way to handle 3 different cases. open to a better option tho!

File: src/homepageExperience/components/steps/Finish.tsx (hunk 2 above)
Comment (ChitlangeSahas) [reply src/homepageExperience/components/steps/Finish.tsx:97]: ✅

File: src/homepageExperience/components/steps/Finish.tsx [hunk 4]
```diff
@@ -61,6 +61,10 @@ const fireConfetti = () => {
   })
 }
 
+const handleNextStepEvent = (nextStepName: string) => {
+  event(`firstMile.nextSteps.${nextStepName}.clicked`)
```
Comment (hoorayimhelping) [inline src/homepageExperience/components/steps/Finish.tsx:65]: let's also add the `props.wizardEventName` here, so something like:

```ts
event(`firstMile.${props.wizardEventName}.nextSteps.${nextStepName}.clicked`)
```

**Review by hoorayimhelping — APPROVED** [review]

File: src/homepageExperience/components/steps/Finish.tsx (hunk 4 above)
Comment (ChitlangeSahas) [reply src/homepageExperience/components/steps/Finish.tsx]: oops, added now.

**Pull request merged**